Elizabeth_Theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<@liferay_util["include"] page=top_head_include />
+
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+<#--  POSITION OF HEADER FIXED HERE!!!!!!!!  -->
+<header class="${header_css_class}" style="position:fixed">
+
+<#--  <header class="${header_css_class}">  -->
+	<div class="container-fluid" id="banner" role="banner">
+		<a class="${logo_css_class}" href="${site_default_url}" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />">
+			<img alt="${logo_description}" height="${site_logo_height}" src="${site_logo}" />
+			<#--  Change the site name  -->
+			<#if show_site_name>
+				HAMBURGUES
```

Comment:
The site name should be configurable by the end user rather than being hardcoded.